src/content/en/updates/2017/06/budget-api.md

```diff
@@ -11,14 +11,14 @@ description: The Budget API allows developers to perform background actions with
 
 The Push Messaging API enables us to send notifications to a user even when the
 browser is closed. Many developers want to be able to use this messaging to
-update and synchornise content without the browser being open, but the API has
+update and synchronize content without the browser being open, but the API has
```

Comment:
Hey Pete, thanks for the feedback. Two things.

1. The word is completely misspelled. (sync**hor**nised)
2. Elsewhere in the document, the American spelling is used; I feel like it should at least be consistent on a per-document basis.